src/Lotto.jsx

```diff
@@ -1,46 +1,81 @@
-import { useState, useEffect } from 'react';
-import './app.css';
-import LottoInfo from './components/LottoInfo';
-import TicketBox from './components/BuyTickets';
+import { useState, useEffect } from "react";
+import "./app.css";
+import LottoInfo from "./components/LottoInfo";
+import TicketBox from "./components/BuyTickets";
+import Modal from "./components/Modal";
 
 function App() {
+   const [price, setPrice] = useState(null);
+   const [randomArray, setRandomArray] = useState([]);
+   const [winner, setWinner] = useState([0]);
+   const [popupIs, setPopupIs] = useState(false);
```

Comment:
```suggestion
   const [isStatisticModalOpen, setStatisticModalOpen] = useState(false);
```
어떤 행동을 하는지 유추할 수 있는 변수명을 지읍시다 😢